src/local-history/local-history-tree-provider.ts

```diff
@@ -64,9 +65,20 @@ export class LocalHistoryTreeProvider implements vscode.TreeDataProvider<Revisio
             revisions.push(new Revision(item.timestamp, item.uri, {
                 command: 'extension.openRevisionInDiff',
                 title: '',
-                arguments: [item.uri]
+                arguments: [item.uri],
+                tooltip: item.uri
             }));
         });
         return revisions;
     }
+
+    /**
+     * Get the human-readable message of the filename.
+     * @returns the human-readable message of the filename.
+     */
+    getTreeViewMessage(): string {
+        return vscode.window.activeTextEditor
+            ? path.basename(vscode.window.activeTextEditor.document.fileName)
+            : '';
```

Comment:
Should we display a message (ex. `No active editor detected`) when there is no opened editor?